apps/admin_app/assets/css/promotions.css

```diff
@@ -0,0 +1,169 @@
+.App {
+    text-align: center;
+  }
+  
+  .App-logo {
+    animation: App-logo-spin infinite 20s linear;
+    height: 40vmin;
+  }
+  
+  .App-header {
+    background-color: #282c34;
+    min-height: 100vh;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    font-size: calc(10px + 2vmin);
+    color: white;
+  }
+  
+  .App-link {
+    color: #61dafb;
+  }
+  
+  .form .formrow{
+    text-align: left;
+    margin:25px auto;
+    max-width: 400px;
+  }
+  
+
+
+  .topnav{
+    background-color: rgba(255,255,255,1);
+    overflow:hidden;
+    margin-top: 20px; 
+    height: 70px;
+  }
+
+  .topnav a{
+    font-size: 30px;
+    float:right;
+    padding:10px 50px 10px 50px;
+  }
+
+  .topnav button{
+    font-size: 30px;
+    float:right;
+    padding:10px 50px 10px 50px;
+    z-index: 2;
+    background: none;
+    background-color: lightslategray;
+  }
+
+  .topnav button:hover{
+    cursor: pointer;
+  }
+
+  .info{
+    font: 20px sans-serif;
+    margin: 70px auto 0px 200px;
+  }
+
+  .rulesandactions{
```

Comment:
@pkrawat1 `rulesandactions ` doesn't look like the best name to me. what in your opinion should be a good name here. how about `rules-and-actions` or something else?

Reply:
rules-and-actions sounds good, will change it